examples/kafka/kafka-ephemeral.yaml

```diff
@@ -14,6 +14,19 @@ spec:
       transaction.state.log.min.isr: 2
     storage:
       type: ephemeral
+    metrics:
```

Comment:
This change should not be here.

Reply:
Even the logging part should not be there. (see the current master versions: https://github.com/strimzi/strimzi-kafka-operator/blob/master/examples/kafka/kafka-ephemeral.yaml). If you need the logging for some testing, you should have a resource somewhere else than in examples ()ion ST resources or something like that..